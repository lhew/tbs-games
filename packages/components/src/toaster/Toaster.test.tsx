import React from 'react';
import TestRenderer from 'react-test-renderer'; // ES6
import Toaster from '.';

test('Toaster scenarios', () => {
  const defaultBtn = TestRenderer.create(<Toaster>Hello</Toaster>);
  expect(defaultBtn).toMatchSnapshot();
});
