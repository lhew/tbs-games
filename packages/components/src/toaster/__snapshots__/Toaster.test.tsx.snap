// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Toaster scenarios 1`] = `
<div
  className="sc-fzozJi dyhXEA"
>
  Hello
</div>
`;

exports[`Toaster scenarios 2`] = `
<div
  className="sc-fzozJi dyhXEA"
>
  Styled toaster
</div>
`;

exports[`Toaster scenarios 3`] = `
<div
  className="sc-fzozJi jBYjbj"
>
  Styled opened toaster
</div>
`;
